recipe/meta.yaml

```diff
@@ -10,7 +10,9 @@ package:
 
 source:
   url: https://github.com/openucx/ucx/archive/v{{ ucx_version }}.tar.gz
-  sha256: e26d1696912cc4a005b93cd7fe61a8d8195e6e61bd5ded47fd232b29f6c94b8b
+  sha256: 9bef31aed0e28bf1973d28d74d9ac4f8926c43ca3b7010bd22a084e164e31b71
+  patches:
+    - 8060.patch  # Likely unnecessary after 1.12.1
```

Comment:
Curious, why was this patch needed?

Reply:
The patch fixes an issue in 1.12.1: https://github.com/openucx/ucx/pull/8060. A fix has been merged, so hopefully the next release won't need that patch anymore.